qiskit_metal/_gui/utility/utils.py

```diff
@@ -0,0 +1,16 @@
+import re
+"""
+Given a filename and a search target, we
+open that file and search
+"""
+
+
+def findProperty(filename, searchTarget):
+    try:
+        with open(filename, 'r') as readfile:
+            filetext = readfile.read()
```

Comment:
Nice, you can also do `my_str = Path('test.md').read_text()` see https://realpython.com/python-pathlib/
this way you dont need open

Reply:
Ypu will need to check if file exists and if read error